### DISCUSSION TITLE
6660 + 7.90 Labor - bekomme aktuell WARNUNG "Filesystem too big"

### DISCUSSION THREAD

**Discussion by Er4twXz** [Freetz-NG/freetz-ng#1005] [Allgemein] [answered]
Mit dem aktuellen Stand bekomme ich beim bauen eines Images mit der Laborversion 7.90 ohne zusätzliche Software:

```
copying kernel image
  kernel image size: 9.0 MB
copying filesystem image
  filesystem image size: 35.8 MB, max 19.0 MB, free -16.8 MB (-17666048 bytes)
WARNING: filesystem image is 17666048 bytes too big. See https://freetz-ng.github.io/freetz-ng/wiki/00_FAQ/FAQ.de.html#filesystem-image-too-big for details.
packing uimg file
  uimg file size: 57.9 MB (60683772 bytes)
```

Im Internet habe ich dazu nur gefunden:
https://www.ip-phone-forum.de/threads/freetz-image-immer-too-big.260426/
Wenn neue Versionen einer Bibliothek in Freetz aufgenommen werden kommt es manchmal vor, dass beim Bauen der Firmware dann mehrere Libraries ins Image kommen.

Mit der 7.50 Version passt alles

**Comment by Conan179** [comment]
Zuerst wäre die Daten der Box Hilfreich,
Welche Box hast du?
Was hast du alles rein gewählt?
Kannst du die .config hier hoch laden?

**Comment by Er4twXz** [comment]
Box: 6660
Firmware: Labor 7.90
Keine zusätzliche Software

[my.config.txt](https://github.com/user-attachments/files/15925059/my.config.txt)

Output:

```
STEP 3: PACK/SIGN
  checking for left over version-control-system files
  integrate freetz info file into image
packing var.tar
  checking signature key files
    adding public signature key file
creating filesystem image (SquashFS4_le-xz)
  SquashFS block size: 64 kB (65536 bytes)
copying kernel image
  kernel image size: 9.0 MB
copying filesystem image
  filesystem image size: 35.7 MB, max 19.0 MB, free -16.7 MB (-17547264 bytes)
WARNING: filesystem image is 17547264 bytes too big. See https://freetz-ng.github.io/freetz-ng/wiki/00_FAQ/FAQ.de.html#filesystem-image-too-big for details.
packing uimg file
  uimg file size: 57.8 MB (60560892 bytes)
packing images/6660_07.90.ger-rev113092_labor_freetz-ng-23950-f7bb61f97_20240621-092505.image
  packed image file size: 57.9 MB (60672000 bytes)
signing packed .image file
  signed image file size: 57.9 MB (60682240 bytes)
source firmware: 6660-Cable_de-es-fr-it-nl-pl 252.07.90-BETA rev113092 {GER} [Smart24 P1] (15.05.2024 00:30:18) 
  source image file size: 57.1 MB (59914240 bytes)
done.

FINISHED


```

Es gibt eine neuere Laborversion 7.90-113461 als die die in Freetz verwendet wird -- das Ergebnis ist allerdings das gleiche.


Das Ganze scheint schon mal funktioniert zu haben, da ich auf meiner Fritzbox ein Freetz mit der aktuellen 113461 habe.

**Comment by fda77** [comment]
1. warnung != error  (ungleich)
2.  bei einem error bricht der vorgang ab, bei warnung nicht

Bei den Puma7 Geräten (neuere "Cable") kann aktuell nicht die maximal mögliche Partitionsgrösse ausgelesen werden, da mit neuem FOS (inhaus+labor) es nicht mehr dort steht wo es immer war: https://github.com/Freetz-NG/freetz-ng/blob/master/fwmod#L2059

**Reply by Er4twXz** [reply]
Wenn ich versuche das Image zu flashen kommt:

HINWEIS: Es gibt kein ausführbares Nach-Installationsskript.

burnuimg: [ERROR] avmipc_command: avmipc_stream_open failed (-1)
burnuimg: [ERROR] aicmd_command(pumaglued uimg update): failed
burnuimg: [ERROR] exit(sys error)

FEHLGESCHLAGEN

**Comment by fda77** [comment]
Das sagt mir nichts, aber mit der warnung hat es nichts zu tun. Flash doch über den bootloader.
Es gibt ja heir und da Gerüchte dass bei den uimg es problem beim Flashen über das Webif Probleme gibt
Was wirklich Sache ist kann ich aber nicht sagen da ich so kein Gerät hab

**Reply by Er4twXz** [reply]
Werd ich probieren

tools/push_firmware

muss noch den Rechner übers LAN Kabel verbinden

**Reply by Er4twXz** [reply, marked as answer]
Habe heute einen git pull gemacht und make nochmal ausgeführt -- jetzt kommt keine Warnung mehr :-)

Es gibt seite heute eine neue Labor die allerdings auch einen neue Kernelversion hat. Ich denke ich werde jetzt bis zur 7.90 final warten.

**Comment by fda77** [comment]
Ich hab die Größe fest hinterlegt, es sollten für die puma7 Geräte ~72mb für das Atom-Filesystem sein. Dabei fiel mir auch noch auf dass Puma6 nicht so ganz korrekt war

Solange du keine Kernelmodule brauchst kannst du die Labor ausprobieren.

Hier steht wie man Sourcen von AVM anfordert: https://freetz-ng.github.io/freetz-ng/SUPPORT.html#quellcode
Und nicht abwimmeln lassen, es gibt in der GPL >keine< Ausnahme für AVM! Die winden sich da gerne.
Solltest du eine Ausrede erhalten würde mich ineterssieren welche
